mautic/mautic
# TPROD-334 - MauticCrmBundle - Convert PHP Templates to Twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to `/s/plugins` (Settings -> Plugins)
3. Click on the **Dynamics CRM**
4. Verify the modal renders correctly.

**NOTE**: The only template is used at the bottom of the modal where it has a link that says "Click here to go to the Dynamics CRM plugin documentation"
